test/fast/Unit tests/nvm_supports_xz

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+cleanup () {
+  rm ../../xz-test/{which,command,awk,rm}
+  rmdir ../../xz-test
+}
+die () { echo $@ ; exit 1; }
+
+. ../../../nvm.sh
+
+mkdir ../../xz-test
+touch ../../xz-test/xz
+chmod +x ../../xz-test/xz
+ln -s /usr/bin/which ../../xz-test/which
+ln -s /usr/bin/command ../../xz-test/command
+ln -s /usr/bin/awk ../../xz-test/awk
```

Comment:
I don't think you'd have to do that - you could do `alias xz`, or `xz() { }`, or `which() { }` - other tests use aliases and shell functions to shadow binaries and builtins quite successfully.